modelci/app/readme.md

```diff
@@ -3,7 +3,7 @@
 ## Start server
 
 ```shell script
-uvicorn main:app
+uvicorn modelci.app.main:app --port 8000
```

Comment:
Shall we assume user start the application from the current directory

Reply:
maybe we should use modelci CLI instead

Reply:
I don't think so, this backend app is independent of CLI.